### DISCUSSION TITLE
Give your idea to select the theme and the story

### DISCUSSION THREAD

**Discussion by BrahimHM** [202306-NEA-DZ-FEW/madlibs-fennecs#3] [Announcements]
**let's discuss the theme and the story here**

**Comment by Khalil-NOUI** [comment]
I was thinking about an AI Theme ;
and the reason why it's a global theme,  modern, and contains diverse aspects.....
here are some random expressive pics of what I have in mind :
![th-1325741026](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/133600682/8916a7d4-aae0-4bb1-bbfc-054a0c9ea8b9)
![th-1551773618](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/133600682/4723773d-4010-47e1-93ab-c625832b7bd7)
![th-2293615598](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/133600682/7ee40c29-b47a-49e2-b88d-ce71bd9f2dea)
![th-3750356674](https://github.com/202306-NEA-DZ-FEW/madlibs-fennecs/assets/133600682/d2c2bf01-beab-4e02-8f06-d2239fd806bd)

**Comment by dzc0d3r** [comment]
As we discussed , we made  a decision about  the story and theme and we have a first Draft : 



**The Cosmic Celebration**

Once upon a time, the planets decided to throw a cosmic wild party. Each planet proudly showcased their quirky spacey characteristics. Mercury grooved happily, Venus sparkled with glitter, Mars revealed its sandy awesome deserts. Jupiter entertained with its gigantic storms, Saturn amazed with its shiny rings. Uranus and Neptune paraded their funky colorful shades. Surprisingly, even tiny Pluto managed to surprise everyone.

In the midst of the festivity, Earth shared stories about its watery oceans and bustling busy existence. Every planet had an epiphany; they were uniquely cool in their own distinct way. The celestial soirée lit up the sky, reminding all that when combined, they shaped an astounding stellar unit



We need to make the correct changes and update `story.txt`,  Anyone want to do this ? so we can open an issue and assign team members  ?  



**Finally** 
We need to meet at 20:00 and discuss more details and start dividing the project into pieces .
